schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE animals (
+    id int GENERATED ALWAYS AS IDENTITY,
+    name string(100),
```

Comment:
- [ ] Good job so far, however, I think In PostgreSQL, the string data type is not a valid data type. Instead, you can use VARCHAR or TEXT data types to store character data. please try to fix this